### PR TITLE
Scala & JVM integrations

### PR DESCRIPTION
**Pull request by RichardSmedley**
DOC-4925﻿

### CODE REVIEW

File: modules/project-docs/pages/third-party-integrations.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+= Integrations & Ecosystem
```
Comment (maria-robobug) [inline modules/project-docs/pages/third-party-integrations.adoc:1]: Don't we need a `:description:` here?

File: modules/project-docs/pages/third-party-integrations.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Couchbase also supports integrating with xref:2.4@spark-connector:ROOT:getting-started.adoc.adoc[Spark].
+
+
+include::6.6@sdk:shared:partial$integrations.adoc[tag=important]
+
+Why not take advantage of compatible Java tools? 
+Many dataflow tools integrate with Couchbase, including https://github.com/apache/nifi/tree/main/nifi-nar-bundles/nifi-couchbase-bundle[Apache NiFi], 
+https://wildfly-extras.github.io/wildfly-camel/#_camel_couchbase[Apache Camel],
+and https://github.com/couchbaselabs/flink-connector-couchbase[Apache Flink].
```
Comment (osfameron) [inline modules/project-docs/pages/third-party-integrations.adoc:26]: missing external link markup `^`

**Comment by osfameron** [comment]
👍  lgtm

File: modules/project-docs/pages/third-party-integrations.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/project-docs/pages/third-party-integrations.adoc:1]: Yes, can add in the 1.2 version.  :+1:

File: modules/project-docs/pages/third-party-integrations.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/project-docs/pages/third-party-integrations.adoc:26]: Got it - will merge this (older version, lower traffic) as is - and change in 1.2

**Comment by RichardSmedley** [comment]
https://github.com/couchbase/docs-sdk-scala/pull/48

**Pull request merged**